brave/browser-laptop
# Sites and wallet now cannot be deleted when payment is in progress

Fixes #14278 

## Submitter Checklist:


## Test Plan:

- Run Brave with appropriate staging environment variables.
- Ensure site can be deleted when a reconcile is not in progress
- Trigger contributions and ensure site cannot be deleted when a reconcile is in progress.


## Reviewer Checklist:


Tests



cc @bradleyrichter for msgbox text and UI/UX. Current message when trying to delete a site from the ledger during contribution:
`A Brave Payment contribution is in progress. Brave Payment data cannot be cleared during this time.`

When deleting from ledgerTable:
<img width="605" alt="screen shot 2018-06-10 at 7 20 06 pm" src="https://user-images.githubusercontent.com/25326067/41208314-29018e64-6cf0-11e8-8afb-0c115faa8f68.png">

In advanced settings:
<img width="705" alt="screen shot 2018-06-10 at 7 19 52 pm" src="https://user-images.githubusercontent.com/25326067/41208321-33b78b7e-6cf0-11e8-800d-5902e79c8aa1.png">
